package proto_ff;

import "yd_fieldoptions.proto";


message rechargeReturnrechargeReturn
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 group = 2[(yd_fieldoptions.field_cname) = "组"];
	optional int32 minLine = 3[(yd_fieldoptions.field_cname) = "激活充值点"];
	optional int32 diamondRRatio = 4[(yd_fieldoptions.field_cname) = "钻石返还比例"];
	optional int32 timeID = 5[(yd_fieldoptions.field_cname) = "时间id"];
	optional int64 mailID = 6[(yd_fieldoptions.field_cname) = "邮件id"];
}


message Sheet_rechargeReturnrechargeReturn
{
	repeated rechargeReturnrechargeReturn rechargeReturnrechargeReturn_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message rechargeReturntime
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 timeType = 2[(yd_fieldoptions.field_cname) = "时间类型"];
	optional string timeA = 3[(yd_fieldoptions.field_cname) = "开始时间", (yd_fieldoptions.field_bufsize) = 60];
	optional string timeB = 4[(yd_fieldoptions.field_cname) = "结束时间", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_rechargeReturntime
{
	repeated rechargeReturntime rechargeReturntime_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message rechargeReturncontinuousReturn
{
	optional int32 Gear = 1[(yd_fieldoptions.field_cname) = "天数档位"];
	optional int64 boxid = 2[(yd_fieldoptions.field_cname) = "发送奖励"];
}


message Sheet_rechargeReturncontinuousReturn
{
	repeated rechargeReturncontinuousReturn rechargeReturncontinuousReturn_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message rechargeReturnrechargeTrigger
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 timeType = 2[(yd_fieldoptions.field_cname) = "时间类型"];
	optional string timeA = 3[(yd_fieldoptions.field_cname) = "开始时间", (yd_fieldoptions.field_bufsize) = 60];
	optional string timeB = 4[(yd_fieldoptions.field_cname) = "结束时间", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 reset = 5[(yd_fieldoptions.field_cname) = "是否每日重置"];
	optional int32 triggerType = 6[(yd_fieldoptions.field_cname) = "触发类型"];
	optional string TriggerValue = 7[(yd_fieldoptions.field_cname) = "触发值", (yd_fieldoptions.field_bufsize) = 180];
	optional int64 boxId = 8[(yd_fieldoptions.field_cname) = "发送奖励"];
	optional int64 mailId = 9[(yd_fieldoptions.field_cname) = "邮件id"];
}


message Sheet_rechargeReturnrechargeTrigger
{
	repeated rechargeReturnrechargeTrigger rechargeReturnrechargeTrigger_List = 1[(yd_fieldoptions.field_arysize)=60];
}

message rechargeReturnconstant
{
	optional int64 constantid = 1[(yd_fieldoptions.field_cname) = "配置id"];
	optional int64 constantdata = 2[(yd_fieldoptions.field_cname) = "数据"];
	optional string stringdata = 3[(yd_fieldoptions.field_cname) = "字符串数据", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_rechargeReturnconstant
{
	repeated rechargeReturnconstant rechargeReturnconstant_List = 1[(yd_fieldoptions.field_arysize)=20];
}
